src/components/error/ErrorBoundary.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import ErrorIcon from '../../icons/GlyphSmallCancel'
+
+class ErrorBoundary extends React.Component {
+  state = {
+    hasError: false
+  }
+
+  static defaultProps = {
+    fallback: ErrorIcon
+  }
+
+  componentDidCatch (error, info) {
+    this.setState({ hasError: true })
+    console.log(error)
```

Comment:
A wild log appeared.

Reply:
That one is deliberate, It logs the error that caused a component subtree to fail to render

Reply:
Yup, my bad 👍